quixstreams/processing/pausing.py

```diff
@@ -18,91 +18,75 @@ class PausingManager:
     the timeout is elapsed.
     """
 
-    _paused_tps: Dict[Tuple[str, int], float]
-
-    def __init__(self, consumer: BaseConsumer):
+    def __init__(self, consumer: BaseConsumer, topic_manager: TopicManager):
         self._consumer = consumer
-        self._paused_tps = {}
-        self._next_resume_at = _MAX_FLOAT
+        self._topic_manager = topic_manager
+        self._resume_at = _MAX_FLOAT
```

Comment:
```suggestion
        self.reset()
```
Plus, I'd move the reset method to the top right under the __init__.